src/ProgressOnderwijsUtils/Html/HtmlToStringExtensions.cs

```diff
@@ -1,7 +1,20 @@
 using System.Buffers;
+using System.IO.Pipelines;
 
 namespace ProgressOnderwijsUtils.Html;
 
+readonly record struct PipeSink(Encoding Encoding, IBufferWriter<byte> writer) : IStringSink
+{
+    public void AppendText(ReadOnlySpan<char> text)
+        => Encoding.GetBytes(text, writer);
+}
+
+readonly record struct WriterSink(StreamWriter writer) : IStringSink
```

Comment:
Would'nt a PipeWriter make more sense in this context?

Reply:
I added both, mostly to be able to compare.